spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -1278,6 +1281,7 @@ public void close() {
     if (numReferenceOfSparkContext.decrementAndGet() == 0) {
       sc.stop();
       sc = null;
+      sparkSession = null;
       if (classServer != null) {
```

Comment:
set sparkSession as null, so that it will be created again if the interpreter is scoped